RELEASE

```diff
@@ -2,10 +2,10 @@
 
 GEOSX version 0.2.0
 
- Copyright (c) 2018-2020 Lawrence Livermore National Security LLC
- Copyright (c) 2018-2020 The Board of Trustees of the Leland Stanford Junior University
- Copyright (c) 2018-2020 TotalEnergies
- Copyright (c) 2019-     GEOSX Contributors
+ Copyright (c) 2018-2024 Lawrence Livermore National Security LLC
+ Copyright (c) 2018-2024 The Board of Trustees of the Leland Stanford Junior University
+ Copyright (c) 2018-2024 TotalEnergies
+ Copyright (c) 2019-     GEOS/GEOSX Contributors
```

Comment:
add Chevron?

Reply:
actually. I think I need to delete these files